src/Pact/Native.hs

```diff
@@ -559,16 +559,18 @@ hexStringToIntegerFun :: RNativeFun e
 hexStringToIntegerFun i as =
```

Comment:
Minor nit: please follow the style of naming the function as close to the camel-cased lisp as possible, ie `hexStrToInt`. The `Fun` suffix adds no information.

Reply:
Along those lines, maybe rename `stringHashNativeFun` accordingly, your call.

Reply:
Changed